common/sepolicy/vendor/hal_bootctl_default.te

```diff
@@ -0,0 +1 @@
+allow hal_bootctl_default sysfs_dt_firmware_android:dir { read open };
```

Comment:
@MaksymPrymierov ,

Should we upstream this?
https://cs.android.com/android/platform/superproject/+/master:system/sepolicy/vendor/hal_bootctl_default.te;l=9;drc=9853c7522d3e81d10c3410b1ea272b385905072e